### ISSUE TITLE
Heap status foreground colour is hard to read

### ISSUE THREAD

**Issue by ingomohr** [AObuchow/Eclipse-Modern-Dark-Theme#38]
<img width="161" alt="Screenshot 2020-06-12 at 00 11 50" src="https://user-images.githubusercontent.com/2838592/84444159-62e13700-ac41-11ea-92ab-2b037bf64264.png">

Maybe we could give it some read and green(ish) as well :)

**Comment by AObuchow** [comment]
Nice catch, I usually have the heap status hidden so I wouldn’t have noticed this :) good suggestion!

**Comment by ingomohr** [comment]
> Nice catch, I usually have the heap status hidden so I wouldn’t have noticed this :) good suggestion!

Yup, me too. I just switched to another workspace where I have it visible... and there it was ;)

**Comment by AObuchow** [comment]
It seems that the colors used are hardcoded to some extent, see [HeapStatus.java](https://git.eclipse.org/c/platform/eclipse.platform.ui.git/tree/bundles/org.eclipse.ui.workbench/Eclipse%20UI/org/eclipse/ui/internal/HeapStatus.java#n133).

An upstream bug should be made about this.

**Comment by AObuchow** [comment]
Upstream bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=564710
